lib/chat_api_web/controllers/note_controller.ex

```diff
@@ -0,0 +1,51 @@
+defmodule ChatApiWeb.NoteController do
+  use ChatApiWeb, :controller
+
+  alias ChatApi.Notes
+  alias ChatApi.Notes.Note
+
+  action_fallback ChatApiWeb.FallbackController
+
+  @spec index(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def index(conn, _params) do
+    notes = Notes.list_notes()
```

Comment:
this method currently fetches all of the notes in the entire database -- let's make sure we filter by `customer_id` and/or `account_id`

(remember, we can get the `account_id` from `conn.assigns.current_user.account_id`)

Reply:
Great, this clears up my main question

Reply:
Nice call!